src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -125,6 +131,8 @@ def run(self, user_config: dict, sbom_path: str = "", skip_deps: bool = False) -
             The path to the SBOM.
         skip_deps : bool
             Flag to skip dependency resolution.
+        prov_payload : InToToPayload
```

Comment:
```suggestion
        prov_payload : InToToPayload | None
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/654/commits/dcb5386ce6e82970dad399c2285cfa9894bd185a